### DISCUSSION TITLE
Mesh Resolution

### DISCUSSION THREAD

**Discussion by lekhachoa** [microsoft/MixedRealityToolkit-Unity#10354] [General]
Hello,
Just a question: How much is the max Resolution of Mesh/Depth Sensor in Hololens 2. Can I set/push or do something to push every Meshes Resolution to 1 cm.  I did try changing LevelOfDetail, and TrianglesPerCubicMeter but the mesh picture is not good enough

**Comment by polar-kev** [comment]
Hi @lekhachoa, Are you using MRTK? The highest resolution mesh can be accessed via SceneUnderstanding APIs using SceneUnderstanding.SceneMeshLevelOfDetail.Unlimited, as seen in this sample (FYI this sample does not use MRTK):

- https://github.com/microsoft/MixedReality-SceneUnderstanding-Samples/search?q=unlimited
- https://github.com/microsoft/MixedReality-SceneUnderstanding-Samples

It is not going to be 1cm, but it is the highest quality and will be higher than the real time mesh LOD. It will take longer to compute however, I can point you to the scene understanding team for assistance there.

**Reply by lekhachoa** [reply]
Thank you very much indeed
And sorry for the late reply.
So, what is the highest resolution could I get from SceneUnderstanding? Could I use Hololens 2 to scan an object e.g a small machine, machine part in multiple times (to get a dense point cloud, which is supposed to provide more detail for mesh generation)?
